eslint.config.mjs

```diff
@@ -4,4 +4,8 @@ export default hmppsConfig({
   extraIgnorePaths: ['test_results'],
   extraPathsAllowingDevDependencies: ['playwright.config.ts'],
   extraGlobals: { $: false },
```

Comment:
To make Jquery accessible

Reply:
it means you can remove `extraGlobals` now.